src/encoded/types/dataset.py

```diff
@@ -33,14 +33,26 @@ def item_is_revoked(request, path):
 
 
 def calculate_assembly(request, files_list, status):
+    assembly = set()
+    viewable_file_status = ['released','in progress']
+
+    for path in files_list:
+        properties = request.embed(path, '@@object?skip_calculated=true')
+        if properties['status'] in viewable_file_status:
+            if 'assembly' in properties:
+                assembly.add(properties['assembly'])
+    return list(assembly)
+
+
+def calculate_visualize_assembly(request, files_list, status):
     assembly = set()
     viewable_file_formats = ['bigWig', 'bigBed']
     viewable_file_status = ['released','in progress']
 
     for path in files_list:
         properties = request.embed(path, '@@object?skip_calculated=true')
         if properties['file_format'] in viewable_file_formats and \
-                properties['status'] in viewable_file_status:
```

Comment:
Needs a JIRA Ticket number in the PR name.  And your github profile is not showing up in the commit, we'll need to fix that on your local repo.